modules/ROOT/pages/release-notes.adoc

```diff
@@ -23,6 +23,14 @@ These credentials can be consulted and modified:
 * in the <BONITA>/server/conf/tomcat-users.xml file in a *Bonita Tomcat bundle*
 * using xref:bonita-docker-installation.adoc#_monitoring_username[specific Environment parameters] when starting a *Bonita Docker image*
 
+
+=== Dynamic authorization checking active by default (Subscription editions)
+
+xref:identity:rest-api-authorization.adoc#dynamic_authorization[Dynamic authorization checking] is now only available in the *Enterprise* editions, and is now *active by default* for new installations. +
```

Comment:
You use 'is now' twice in a few words, i suggest to find an other way to say it to avoid this repetition. 

Maybe just remove the second now? : 
```suggestion
xref:identity:rest-api-authorization.adoc#dynamic_authorization[Dynamic authorization checking] is now only available in the *Enterprise* editions, and is*active by default* for new installations. +
```

Reply:
True
Thanks